x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -168,6 +271,7 @@ export async function getAgentsByKuery(
   };
 }
 
+// TODO instead of query all at once, do bulk action in batches
```

Comment:
Is it worth creating a ticket for this TODO ?

Reply:
removed